bundles/pom.xml

```diff
@@ -768,4 +769,4 @@
     </profile>
   </profiles>
 
-</project>
+</project>
```

Comment:
Revert this, as this is causing the build error. it is outside of the bundle directory and therefore you don't experience this locally when building from the project subfolder.

Reply:
done